Jenkinsfile

```diff
@@ -272,8 +272,9 @@ node {
       'car tests': {
         pcStage("car tests") {
           sh label: "build", script: "selfdrive/manager/build.py"
-          sh label: "run car tests", script: "cd selfdrive/car/tests && MAX_EXAMPLES=300 INTERNAL_SEG_CNT=300 FILEREADER_CACHE=1 \
-              INTERNAL_SEG_LIST=selfdrive/car/tests/test_models_segs.txt pytest test_models.py test_car_interfaces.py"
+          sh label: "test_models", script: "INTERNAL_SEG_CNT=300 FILEREADER_CACHE=1 INTERNAL_SEG_LIST=selfdrive/car/tests/test_models_segs.txt \
+                                            pytest selfdrive/car/tests/test_models.py"
+          sh label: "test_car_interfaces", script: "MAX_EXAMPLES=300 pytest selfdrive/car/tests/test_car_interfaces.py"
```

Comment:
Why do this? I combined them to speed this stage up

Reply:
The high memory usage seen when running both files together is caused by the `Controls` class disabling gc. Running both, it used to end up at +25GB usage with -n32, now it ends at +7GB which is what it shoots to in the beginning of the test, so it should be stable:
 https://github.com/commaai/openpilot/blob/8f5dcb30199faf0e1e89f0b6ad6a3af3880afc1e/selfdrive/car/tests/test_models.py#L413

Can you disable that line and revert this?

Reply:
nice catch. we should just reenable gc in the pytest fixture